compose-example/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# The jupyterhub/jupyterhub instance is quite bulky... A stripped down version
+# can be built from the python:3.x-slim images.
+#FROM jupyterhub/jupyterhub
+FROM python:3.11-slim
```

Comment:
Is it possible to build on top of the standard JupyterHub image, as in the other example?
https://github.com/jupyterhub/jupyterhub-deploy-docker/blob/89a29629016643c3e6c0454fb4f47d03f7c06f0e/basic-example/Dockerfile.jupyterhub#L4

Reply:
Sure, no problem